src/nocmodl/kinetic.cpp

```diff
@@ -652,7 +645,7 @@ void kinetic_implicit(Symbol* fun, const char* dt, const char* mname) {
     Item* qv;
     Reaction* r1;
     Rlist *rlst, *clst;
-    int i, nstate, flag, sparsedec, firsttrans, firsttrans1;
```

Comment:
That's 200 lines of scope with macros and `#if` directives. No clue if it's used or not.

Reply:
I did a grep in the full repository and this name never appear